source/eventcore/drivers/posix/driver.d

```diff
@@ -124,7 +124,14 @@ final class PosixEventDriver(Loop : PosixEventLoop) : EventDriver {
 			{
 				if (!hasPrintedHeader && (hasPrintedHeader = true) == true)
 					print("Warning (thread: %s): leaking eventcore driver because there are still active handles", getThreadName());
-				print("	  FD %s (%s)", id, s.specific.kind);
+				print("  FD %s (%s)", id, s.specific.kind);
+				debug
```

Comment:
Probably needs to be its own debug version, e.g. `debug (EventCoreLeakTrace)`
Then:
```D
debug (EventCoreLeakTrace) { /* ... */ }
else
    print("Use '-debug=EventCoreLeakTrace' to show where the instantiation happened");
```